tests/parsers/test_pp.py

```diff
@@ -141,7 +141,10 @@ def test_pp_default_1d(
         'data_array': data_array,
         'coords_array': coords_array
     },
-                         default_tolerance=dict(atol=0, rtol=1e-18))
+                         default_tolerance={
+                             'atol': 0,
+                             'rtol': 1e-18
+                         })
```

Comment:
Holy indentation batman! I think if you manually indent this, `yapf` should accept it just fine. This is craziness. Same for the other occurrences of this keyword in the tests.

Reply:
`yapf` was being very stubborn, but it seems adding the first input argument as a keyword did the trick. 😎